dev/tasks/macros.jinja

```diff
@@ -131,16 +131,14 @@ on:
 {% endmacro %}
 
 {%- macro github_upload_gemfury(pattern) -%}
-  {%- if arrow.is_default_branch() -%}
   - name: Upload package to Gemfury
     shell: bash
     run: |
+      gem install --user-install gemfury
       path=$(ls {{ pattern }})
-      curl -F "package=@${path}" https://${CROSSBOW_GEMFURY_TOKEN}@push.fury.io/${CROSSBOW_GEMFURY_ORG}/
+      ruby arrow/ci/scripts/gemfury_upload.rb ${path}
```

Comment:
How about using `fury` command installed by `gem install --user-install gemfury` instead of creating a custom command?

```shell
PATH=$(ruby -r rubygems -e 'puts Gem.user_dir')/bin:$PATH
gem install --user-install gemfury
fury \
  --api-token=${CROSSBOW_GEMFURY_TOKEN} \
  --as=${CROSSBOW_GEMFURY_ORG} \
  {{ pattern }}
```